apps/studio/components/layouts/AdvisorsLayout/Advisors.Commands.tsx

```diff
@@ -0,0 +1,28 @@
+import { useParams } from 'common'
```

Comment:
can remove this guy now?

Reply:
I think we should keep right? This is just the navigation item (I removed the one that shows the lints)